bin/docker-rails

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+set -e
+
+rm -rf /app/tmp/pids/server.pid
+
+bin/rails server --port 3000 --binding 0.0.0.0
```

Comment:
Same question here. --- I think my question actually doesn't matter, as these are the ports exposed in the container vs running the app locally. But just wanted to note that `inkind-volunteer` can not run without this app, `inkind-admin`.

Reply:
same as above, this is internal to container stuff and bound to rails defaults, outside of container it is configured to be port 3001 by default